package.json

```diff
@@ -50,16 +50,23 @@
     "index.js",
     "lib"
   ],
-  "main": "index.js",
   "bin": "./bin/remarkable.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
   "engines": {
     "node": ">= 0.10.0"
   },
   "scripts": {
-    "test": "make test",
-    "test-ci": "nyc mocha -r esm -R spec --bail",
+    "build": "make rollup",
+    "test": "make rollup && make test",
```

Comment:
was Rollup just added here to make sure the build process succeeds?

Reply:
bin/remarkable.js requires built version. This is necessary to make cli test pass.